AlleleParseError was moved to the top-level mhcnames package.

Author: Emik Lin (HKUMed CPOS)
--- a/mhctools/base_commandline_predictor.py
+++ b/mhctools/base_commandline_predictor.py
@@ -20,8 +20,7 @@ import tempfile
 
 from six import string_types
 from typechecks import require_string, require_integer, require_iterable_of
-from mhcnames import normalize_allele_name
-from mhcnames.parsing_helpers import AlleleParseError
+from mhcnames import AlleleParseError, normalize_allele_name
 
 
 from .base_predictor import BasePredictor
